src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -62,16 +61,12 @@ class SignInModal extends React.Component {
             <div className="columns small-12">
               <div className="form-warning-banner">
                 <AlertBox
-                  headline={`Some ${siteName} tools and features may not be working as expected`}
+                  headline={`Some VA.gov tools and features may not be working as expected`}
```

Comment:
Can just be `headline="Some VA.gov tools and features may not be working as expected"`.